samza-core/src/main/java/org/apache/samza/container/RunLoopFactory.java

```diff
@@ -69,6 +69,8 @@ public static Runnable createRunLoop(scala.collection.immutable.Map<TaskName, Ru
 
     log.info("Got current run Id: {}.", runId);
 
+    log.info("Got isHighLevelApiJob flag: {}.", isHighLevelApiJob);
```

Comment:
Nit: replace `Got isHighLevelApiJob flag: {}` to something meaningful instead of exposing the field/accessor name.